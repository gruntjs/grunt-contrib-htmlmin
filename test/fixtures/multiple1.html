<!doctype html>
<html>
  <body>
    <div>first</div>
    <!-- Hello world -->
  </body>
</html>
