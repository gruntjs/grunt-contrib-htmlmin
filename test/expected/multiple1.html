<!doctype html><html><body><div>first</div></body></html>